tensorpack/dataflow/image.py

```diff
@@ -57,7 +68,7 @@ class AugmentImageComponent(MapDataComponent):
     """
     Apply image augmentors on 1 image component.
     """
-    def __init__(self, ds, augmentors, index=0, copy=True):
+    def __init__(self, ds, augmentors, index=0, copy=True, allow_exceptions=False):
```

Comment:
The name `allow_exceptions` confused me at the first look. The current code is saying exception will be raised when exception is "not allowed", which is weird.
Maybe use "hide_exceptions" or "catch_exceptions"?

Reply:
-> catch_exceptions